msg_helpers_test.go

```diff
@@ -397,12 +397,16 @@ func TestUnpackDataAplPrefix_Errors(t *testing.T) {
 			"extra bits set",
 			[]byte{0x00, 0x01, 22, 0x03, 192, 0, 2},
 		},
+		{
+			"afdlen invalid",
+			[]byte{0x00, 0x01, 22, 0x05, 192, 0, 2, 0, 0},
+		},
 	}
-	for _, tt := range tests {
+	for i, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			_, _, err := unpackDataAplPrefix(tt.wire, 0)
 			if err == nil {
-				t.Fatal("expected error, got none")
+				t.Fatalf("[%d] sexpected error, got none", i)
```

Comment:
name should be enough here, so please revert this, so we have tiny, well contained PR